PukingPredator/Assets/Scripts/GameInput.cs

```diff
@@ -10,6 +10,7 @@ public class GameInput : MonoBehaviour
     private PlayerInput playerInput;
     public EventHandler onInteractAction;
```

Comment:
what interactions do we have? if it's just eating, could we rename this to `onEatAction`?